.travis.yml

```diff
@@ -38,6 +40,11 @@ script:
   - cd importers/openaddresses && cargo check
   - cargo test
   - cd ../..
-  - cd tools && cargo check
+  # Check for deduplicator
+  - cd deduplicator && cargo check
   - cargo test
   - cd ..
+  # Run OSM importer and a complete deduplication
+  - wget https://download.geofabrik.de/europe/luxembourg-latest.osm.pbf
```

Comment:
I would prefer to reuse the .pbf file that already exists in this repo, instead of depending on Geofabrik servers.